docker-compose.development.yml

```diff
@@ -1,5 +1,3 @@
-version: "3"
-
```

Comment:
This is just the log system lying to you, compose definitely does support versions. Or at least it did. I remember getting this error sometimes, but it was caused by something else.